### ISSUE TITLE
<utrecht-digid-logo> has no document method of providing localized text alternative to image

### ISSUE THREAD

**Issue by Robbert** [nl-design-system/utrecht#2011]
No text alternative is possible currently.

A builtin text alternative woud be problematic too (`aria-label="DigiD logo"`), since the user should be able to provide localized text alternative.
